examples/practice-typescript/src/components/TableBody/index.tsx

```diff
@@ -0,0 +1,30 @@
+import TableRow from 'components/TableRow';
+import User from 'scripts/types/user';
+
+interface TableListProps {
+  children: User[];
+}
+
+const editUser = () => console.log('edit');
+
+const deleteUser = () => console.log('delete');
+
+const TableBody = ({ children = [] }: TableListProps) => {
```

Comment:
This table only work for data that is User, so that we have 2 options here
- Rename the component for specific user data
- Make it generic

Reply:
Fixed, Make it generic